app/views/event_ratings/show.html.haml

```diff
@@ -75,6 +75,17 @@
         %fieldset.inputs
           = f.input :rating, as: :rating
           = f.input :comment, input_html: {rows: 3}
+          = f.simple_fields_for :review_scores do |ff|
+            = ff.input :score,
+              as: :radio_buttons,
+              collection: [[t('events_module.not_applicable'), '0'], ['1', '1'], ['2', '2'], ['3', '3'], ['4', '4'], ['5', '5']], 
```

Comment:
Does the zero value mean that you can undo a rating?

Reply:
Yes. The value 0 is displayed as N/A when rating; and as a "blank" when looking at the rating summary. It does not taken account when calculating average review.

![image](https://user-images.githubusercontent.com/20379005/67691714-b69b5a80-f9a7-11e9-912f-d46a30a03028.png)